Source/FlowEditor/Private/Graph/Nodes/FlowGraphNode.cpp

```diff
@@ -553,12 +566,30 @@ void UFlowGraphNode::CreateAttachAddOnSubMenu(UToolMenu* Menu, UEdGraph* Graph)
 
 bool UFlowGraphNode::CanUserDeleteNode() const
 {
-	return NodeInstance ? NodeInstance->bCanDelete : Super::CanUserDeleteNode();
+	if (NodeInstance)
+	{
+		return NodeInstance->bCanDelete;
+	}
+	else if (NodeInstanceClass)
+	{
+		UFlowNodeBase* Node = NodeInstanceClass->GetDefaultObject<UFlowNodeBase>();
+		return Node->bCanDelete;
+	}
+	return Super::CanUserDeleteNode();
 }
 
 bool UFlowGraphNode::CanDuplicateNode() const
 {
-	return NodeInstance ? NodeInstance->bCanDuplicate : Super::CanDuplicateNode();
+	if (NodeInstance)
+	{
+		return NodeInstance->bCanDuplicate;
+	}
+	else if (NodeInstanceClass)
+	{
+		UFlowNodeBase* Node = NodeInstanceClass->GetDefaultObject<UFlowNodeBase>();
+		return Node->bCanDuplicate;
+	}
+	return Super::CanDuplicateNode();
```

Comment:
Do we need to allow duplicating node if NodeInstance and NodeInstanceClass are invalid? This node is clearly malformed in this case. Maybe just return false?